teamcity: failed tests on master: acceptance/TestNodeRestart

The following tests appear to have failed:

[#43011](https://teamcity.cockroachdb.com/viewLog.html?buildId=43011):

```
--- FAIL: acceptance/TestNodeRestart (56.143s)
	testing.go:117: acceptance/util_test.go:404, condition failed to evaluate within 45s: failed to send RPC: rpc error: code = 1 desc = context canceled

```
Please assign, take a look and update the issue accordingly.

--- a/pkg/acceptance/cluster/localcluster.go
+++ b/pkg/acceptance/cluster/localcluster.go
@@ -41,6 +41,8 @@ import (
 	"github.com/pkg/errors"
 	"golang.org/x/net/context"
 
+	"sync"
+
 	"github.com/cockroachdb/cockroach/pkg/base"
 	"github.com/cockroachdb/cockroach/pkg/config"
 	roachClient "github.com/cockroachdb/cockroach/pkg/internal/client"
@@ -406,6 +408,8 @@ func (l *LocalCluster) initCluster() {
 	maybePanic(c.Wait())
 }
 
+// createRoach creates the docker container for a testNode. It may be called in
+// parallel to start many nodes at once, and thus should remain threadsafe.
 func (l *LocalCluster) createRoach(node *testNode, vols *Container, env []string, cmd ...string) {
 	l.panicOnStop()
 
@@ -476,6 +480,8 @@ func (l *LocalCluster) createNodeCerts() {
 		keyLen, nodes))
 }
 
+// startNode starts a Docker container to run testNode. It may be called in
+// parallel to start many nodes at once, and thus should remain threadsafe.
 func (l *LocalCluster) startNode(node *testNode) {
 	cmd := []string{
 		"start",
@@ -651,9 +657,15 @@ func (l *LocalCluster) Start() {
 
 	l.monitorCtx, l.monitorCtxCancelFunc = context.WithCancel(context.Background())
 	go l.monitor()
+	var wg sync.WaitGroup
+	wg.Add(len(l.Nodes))
 	for _, node := range l.Nodes {
-		l.startNode(node)
+		go func(node *testNode) {
+			l.startNode(node)
+			wg.Done()
+		}(node)
 	}
+	wg.Wait()
 }
 
 // Assert drains the Events channel and compares the actual events with those

--- a/pkg/acceptance/util_test.go
+++ b/pkg/acceptance/util_test.go
@@ -313,6 +313,7 @@ func runTestOnConfigs(t *testing.T, testFunc func(*testing.T, cluster.Cluster, c
 	for _, cfg := range cfgs {
 		func() {
 			cluster := StartCluster(t, cfg)
+			log.Infof(context.Background(), "cluster started successfully")
 			defer cluster.AssertAndStop(t)
 			testFunc(t, cluster, cfg)
 		}()
@@ -383,18 +384,17 @@ func StartCluster(t *testing.T, cfg cluster.TestConfig) (c cluster.Cluster) {
 			client, dbStopper := c.NewClient(t, 0)
 			defer dbStopper.Stop()
 
-			var cancel context.CancelFunc
-			ctx, cancel = context.WithTimeout(ctx, 5*time.Second)
+			ctxWithTimeout, cancel := context.WithTimeout(ctx, 5*time.Second)
 			defer cancel()
 
 			var desc roachpb.RangeDescriptor
-			if err := client.GetProto(ctx, keys.RangeDescriptorKey(roachpb.RKeyMin), &desc); err != nil {
+			if err := client.GetProto(ctxWithTimeout, keys.RangeDescriptorKey(roachpb.RKeyMin), &desc); err != nil {
 				return err
 			}
 			foundReplicas := len(desc.Replicas)
 
 			if log.V(1) {
-				log.Infof(ctx, "found %d replicas", foundReplicas)
+				log.Infof(ctxWithTimeout, "found %d replicas", foundReplicas)
 			}
 
 			if foundReplicas < wantedReplicas {

